tests/test_math_core.py

```diff
@@ -193,6 +193,9 @@ def test_square(self):
                                 [0.0, 0.0, 0.5, 0.5],
                                 [0.0, 0.0, 0.0, 1.0]]), coords)
 
+    def test_scale_xyz(self):
+        xyz = jnp.array([0., 0., 0.])
+        assert (jnp.allclose(scale_xyz(xyz), xyz))
```

Comment:
@Maikenov this test is very poor. It cannot catch bug with missed scaling inside the function, for example.